content/notebooks/hydrology/01_Intro.ipynb

```diff
@@ -0,0 +1,712 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">config_dir = "tutorial_data/"
output_dir = "/notebook_dir/writable-workspace/tmp/raven_tutorial_output"
</pre><p>Suggest changing line 3 to read from notebook location and line 4 so that can run from 'read-only' dir once on PAVICS  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/PAVICS-landing/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Changes made. Note that this will require RavenPy 0.13.1 or 0.14.1 and up to work.